js-intermediate.js

```diff
@@ -5,15 +5,30 @@
 
 // 1. Write a function called getFib that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+const getFib = (n) => {
+  let arr = [1,1]
+  for (let i = 2; i < n; i++){
+    arr.push(arr[i-1] + arr[i - 2])
+  }
+  return arr
+}
+
+console.log(getFib(10))
 
 
 // 2. Write a function called oddChecker that takes in an array and returns a new array of only odd numbers.
 
+
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [9, 7, 199, -9]
+// // Expected output: [9, 7, 199, -9]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [7, 23, -823]
+// // Expected output: [7, 23, -823]
+
 
+// Almost there! Need help with checking for types. 
+const oddChecker = (arr) => arr.filter(num => num % 2 > 1 || num % 2 > 0 && num != typeof(String() || num != Boolean()));
+console.log(oddChecker(fullArr1))
+console.log(oddChecker(fullArr2))
 
```

Comment:
You are so close!!
var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
const oddChecker = (arr) => {
  let onlyOdds = arr.filter(num => num % 2 !== 0 && typeof num === "number"
  )
  return onlyOdds
}
console.log(oddChecker(fullArr1))